test/test_restful.py

```diff
@@ -373,12 +373,14 @@ def setUpClass(cls):
         # Allow targeting a specific API version (default latest)
         cls.api_version = "2.0"
 
-        # Set up allowlist bundles. Use invalid exclusion list regex for bad bundle.
```

Comment:
Similar comment here about deprecated testing. We will need to test both API usages for a while. So makes sense to have separate tests for each, in this case test methods rather just a new file.

Reply:
As discussed in the team meeting today, we aren't maintaining any legacy tenant <-> verifier APIs, so we shouldn't need to maintain backwards-compatible tests either.